### PR TITLE
[ISSUE #3649] Method invocation may produce 'NullPointerException'.[CloudEventsProtocolAdaptor]

### LINKED ISSUE #3649: [Enhancement] Method invocation may produce 'NullPointerException'.[CloudEventsProtocolAdaptor]
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/eventmesh/issues?q=is%3Aissue) and found no similar issues.

### Enhancement Request

image
located at:
eventmesh-protocol-plugin/eventmesh-protocol-cloudevents/src/main/java/org/apache/eventmesh/protocol/cloudevents/CloudEventsProtocolAdaptor.java line 121,130

### Describe the solution you'd like

image

### Are you willing to submit PR?

- [ ] Yes I am willing to submit a PR!

### PR DESCRIPTION
**Pull request by kyooosukedn**
Fixes #3649 .

### Modifications

*Added Objects.requireNonNull() in line 121 and 130 as well as the import java.util.Objects.*

### Documentation

- Does this pull request introduce a new feature? (no)
- If yes, how is the feature documented? (not applicable )

### CODE REVIEW

File: eventmesh-protocol-plugin/eventmesh-protocol-cloudevents/src/main/java/org/apache/eventmesh/protocol/cloudevents/CloudEventsProtocolAdaptor.java [hunk 1]
```diff
@@ -127,7 +128,8 @@ public ProtocolTransportObject fromCloudEvent(CloudEvent cloudEvent) throws Prot
                 @Override
                 public Map<String, Object> toMap() {
                     byte[] eventByte =
-                        EventFormatProvider.getInstance().resolveFormat(JsonFormat.CONTENT_TYPE).serialize(cloudEvent);
+                        Objects.requireNonNull(EventFormatProvider.getInstance()
+                            .resolveFormat(JsonFormat.CONTENT_TYPE)).serialize(cloudEvent);
                     map.put("content", new String(eventByte, StandardCharsets.UTF_8));
```
Comment (mxsm) [inline eventmesh-protocol-plugin/eventmesh-protocol-cloudevents/src/main/java/org/apache/eventmesh/protocol/cloudevents/CloudEventsProtocolAdaptor.java:133]: StandardCharsets.UTF_8 replace with Constants.DEFAULT_CHARSET

**Review by Alonexc — APPROVED** [review]

**Comment by Alonexc** [comment]
image
please check checkstyle.

**Comment by Alonexc** [comment]
Download this plugin：
image
Import the checkstyle of the local project and check the box.
image
image
image

**Comment by kyooosukedn** [comment]
@Alonexc @xwm1992  Thank you :)

**Review by xwm1992 — APPROVED** [review]

**Pull request merged**